libr/debug/dreg.c

```diff
@@ -276,10 +279,10 @@ R_API int r_debug_reg_list(RDebug *dbg, int type, int size, int rad, const char
 		n++;
 	}
 beach:
-	if (rad == 'j') {
-		dbg->cb_printf ("}\n");
-	} else if (rad == 'J') {
-		// do nothing
+	if (rad == 'j' || rad == 'J') {
+		pj_end (pj);
+		dbg->cb_printf ("%s\n", pj_string (pj));
+		pj_free (pj);
 	} else if (n > 0 && (rad == 2 || rad == '=') && ((n%cols))) {
```

Comment:
spaces around %

Reply:
Do you mean writing `dbg->cb_printf (" %s \n", pj_string (pj));` instead? But, why and is it necessary for a json string?